pattern_printing/InvertedPyramid.java

```diff
@@ -4,6 +4,7 @@ public class InvertedPyramid {
     public static void main(String[] args) {
       
       Scanner s = new Scanner(System.in);
+         System.out.println("Enter any number for the Pyramid's Height");
       int n = s.nextInt();
       
       int stars = 2 * n -1;
```

Comment:
for(int i=0;i<n;i++){
for(int j=0;j<n;j++){
if(i>=j*2 && i<= n-j*2 ){
System.out.print("*");
}else{
System.out.print(" ");
}


}System.out.println();
}